aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -68,47 +68,70 @@ public class AwsProperties implements Serializable {
    * AwsProperties#S3FILEIO_SSE_TYPE_NONE}.
    *
    * <p>For more details: https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html
+   *
+   * @deprecated will be removed in Iceberg 1.4, use {@link
```

Comment:
nit: I think in other places we typically just mention the Iceberg version without `Iceberg`: `@deprecated will be removed in 1.4.0`